compiler/rustc_type_ir/src/flags.rs

```diff
@@ -1,3 +1,9 @@
+use std::slice;
+
+use crate::inherent::*;
+use crate::visit::Flags;
+use crate::{self as ty, GenericArgKind, InferConst, Interner};
```

Comment:
Can you just prepend `InferConst` and `GenericArgKind` with `ty::` rather than importing them?